ntcore/src/main/native/cpp/NetworkServer.cpp

```diff
@@ -360,6 +361,9 @@ void NetworkServer::LoadPersistent() {
         "could not open persistent file '{}': {} "
         "(this can be ignored if you aren't expecting persistent values)",
         m_persistentFilename, ec.message());
+    // backup file
+    std::filesystem::copy_file(m_persistentFilename,
+                               m_persistentFilename + ".bak");
```

Comment:
By default `copy_file` will report an error if the file already exists- pass `std::filesystem::copy_options::overwrite_existing` to overwrite the old backup file